app/models/listing.rb

```diff
@@ -132,17 +130,6 @@ def set_updates_email_at_to_now
   validates_inclusion_of :valid_until, :allow_nil => :true, :in => DateTime.now..DateTime.now + 7.months
   validates_numericality_of :price_cents, :only_integer => true, :greater_than_or_equal_to => 0, :message => "price must be numeric", :allow_nil => true
 
-  def set_community_visibilities
-    if current_community_id
```

Comment:
What is this and do we still need it? Does some logic now work differently because of removing this before save hook?